demos/demo.html

```diff
@@ -181,18 +171,12 @@ <h2 id="code">Source code and syntax</h2>
 
 <h3>Syntax highlighting</h3>
 
-<p>To trigger syntax highlighting using <a href="https://code.google.com/p/google-code-prettify/">prettify</a>, prefix the <code>&lt;pre&gt;</code> with a prettify inline hint such as <code>&lt;?prettify linenums=1?&gt;</code>, or add the <code>prettify</code> class, as documented in the <a href="https://code.google.com/p/google-code-prettify/wiki/GettingStarted">prettify wifi</a>.  Since prettify auto-loads all its lang files, we use it from the Cloudflare CDN.  Put this in your page to enable support:</p>
-
-<?prettify linenums=1?>
-<pre><code>&lt;script src="//cdnjs.cloudflare.com/ajax/libs/prettify/r298/prettify.js">&lt;/script>
-&lt;script&gt;addEventListener('load', function(event) { prettyPrint(); }, false);&lt;/script&gt;</code></pre>
-
+<p>o-techdocs brings syntax highlighting out of the box using <a href="https://highlightjs.org">Highlight.js</a>.</p>
```

Comment:
Ha, it's in by default. I suggest we build an opt-out mechanism when someone requests it. What do you think?